app/views/users/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= "#{current_user.first_name} #{current_user.last_name}" %></h2>
```

Comment:
@ianroberts131 introduce a feature spec that provides test coverage for the user profile show page, e.g. `user_views_profile_spec.rb`. For now it can just test the logged-in users full name is shown in on the page.

If you'd like to follow TDD, when you next want to add more details to the profile page:

1. add an expectation to the feature spec first (e.g. expect page to display user's city)
2. run the modified feature spec and check it fails due to the added expectation
3. write the code to show the user's city
4. run feature spec again and check it passes